lib/models/uc3100-v2.js

```diff
@@ -46,18 +46,37 @@ module.exports = async function upload3100ImageV2(
         out: debug,
       },
 
-      // Update from TFTP
+      // Update firmware
       {
         expect: 'Command>>',
         timeout,
         send: '1\n', // (1) Firmware
         out: debug,
       },
 
+      // TFTP Settings
+      {
+        expect: 'Command>>',
+        timeout,
+        send: '1\n', // (1) TFTP Settings
+        out: debug,
+      },
+      {
+        expect: 'Command>>',
+        timeout,
+        send: '3\n', // (3) Mgmt port Setting
+        out: debug,
+      },
       {
         expect: '(Enter to abort)',
         timeout,
-        send: '1\n', // 1 - Yes, change the ip address
+        send: `${process.env.TFTP_PORT || 1}\n`, // 2 - Yes, change the port to LAN2
```

Comment:
TFTP_PORT was supposed to be the port that they wanted to use. Is this comment still correct?